src/Build/Collections/RetrievableEntryHashSet/HashSet.cs

```diff
@@ -43,7 +43,7 @@ namespace Microsoft.Build.Collections
 {
     /// <summary>
     /// Implementation notes:
-    /// This uses an array-based implementation similar to <see cref="T:Dictionary{T}" />, using a buckets array
+    /// This uses an array-based implementation similar to <see cref="Dictionary{TKey, TValue}" />, using a buckets array
```

Comment:
I am unsure on this change. Can someone please verify it is correct? `Dictionary{T}` doesn't seem to be a thing.

Reply:
This is correct